OCABundles/schema/bcgov-digital-trust/person/OCABundle.json

```diff
@@ -144,28 +144,26 @@
       },
       {
         "capture_base": "ES3khN7HC1zZyr-DeQ6UbUVyhY_npOrxX_8NO_K4IWH8",
-        "credential_help_text": "",
-        "credential_support_url": "https://www.gov.bc.ca/",
-        "description": "A Canadian schema used for the identity of a person",
-        "digest": "EFqwZey67Xkfl4q_rcYroNzZa0f8SUdIdRLAKf4kFU_U",
-        "issuer": "Government of British Columbia",
-        "issuer_description": "Government of British Columbia",
-        "issuer_url": "https://www.gov.bc.ca/",
-        "language": "en",
-        "name": "Person Credential",
+        "credential_support_url": "https://www2.gov.bc.ca/gov/content?id=87EEAD6D19974459950AA7FF7F60AD54",
+        "description": "Schéma canadien utilisé pour l'identité d'une personne",
+        "digest": "EuBEK7v-Jq5bqW72CmB8jOzxRllFiW3u5KbhT6nMVzfg",
+        "issuer": "Service BC",
+        "issuer_description": "Service BC",
+        "issuer_url": "https://www2.gov.bc.ca/gov/content?id=87EEAD6D19974459950AA7FF7F60AD54",
```

Comment:
This URL points to a web page about the physical BC Services Card plastic credential. It is likely ok for the short term. I'll check with IDIM and the team.